Orange/widgets/utils/itemmodels.py

```diff
@@ -869,8 +869,19 @@ class DomainModel(VariableListModel):
                  ATTRIBUTES)
     PRIMITIVE = (DiscreteVariable, ContinuousVariable)
 
-    def __init__(self, order=SEPARATED, placeholder=None,
+    def __init__(self, order=SEPARATED, separators=True, placeholder=None,
                  valid_types=None, alphabetical=False, skip_hidden_vars=True, **kwargs):
+        """
+
+        :param order: How attributes, metas and classes are ordered and where between
+                      them are separators.
+        :param separators: Separators are ignored when set to False despite being set in order.
+        :param placeholder: The first element.
```

Comment:
Wrong. Placeholder is the text that is shown when no variable is selected (i.e., `None`). It is usually, but not necessarily, the first element.

See: https://github.com/jerneju/orange3/blob/e62c7a6b785dab00f5c619c561d4cb361636bce0/Orange/widgets/utils/itemmodels.py#L888. `None` is inserted at the beginning of `order` only if it's not already there.